5calls/app/src/main/res/layout/activity_about.xml

```diff
@@ -101,6 +101,19 @@
             android:textColor="@color/colorAccent"
             />
 
```

Comment:
Could you put a little TextView above this that says something like "Want to get in touch? Are we not covering an issue we should be?". Exact text is up to you!

That would give it the same flow as the other buttons on the screen -- some introductory text followed by an actionable button.

Otherwise looks great!